# HG changeset patch
# User jwe
# Date 828122802 0
#      Fri Mar 29 18:06:42 1996 +0000
# Node ID a7394e14f4a693eee27713b82356a1085ba9e22a
# Parent  31f4f775b8c6ab07604abc60a42e01e3ea41ab77
[project @ 1996-03-29 18:06:42 by jwe]
Initial revision

diff --git a/src/SLStack-i.cc b/src/SLStack-i.cc
new file mode 100644
--- /dev/null
+++ b/src/SLStack-i.cc
@@ -0,0 +1,34 @@
+/*
+
+Copyright (C) 1996 John W. Eaton
+
+This file is part of Octave.
+
+Octave is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2, or (at your option) any
+later version.
+
+Octave is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with Octave; see the file COPYING.  If not, write to the Free
+Software Foundation, 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+*/
+
+// Instantiate stacks of int values.
+
+compl#include <list>
+#include <stack>
+
+template class stack <list <int> >;
+
+/*
+;;; Local Variables: ***
+;;; mode: C++ ***
+;;; End: ***
+*/
diff --git a/src/SLStack-pc.cc b/src/SLStack-pc.cc
new file mode 100644
--- /dev/null
+++ b/src/SLStack-pc.cc
@@ -0,0 +1,34 @@
+/*
+
+Copyright (C) 1996 John W. Eaton
+
+This file is part of Octave.
+
+Octave is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2, or (at your option) any
+later version.
+
+Octave is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with Octave; see the file COPYING.  If not, write to the Free
+Software Foundation, 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+*/
+
+// Instantiate stacks of char* values.
+
+compl#include <list>
+#include <stack>
+
+template class stack <list <char *> >;
+
+/*
+;;; Local Variables: ***
+;;; mode: C++ ***
+;;; End: ***
+*/
